### PR TITLE
App Details - custom icon & app name

### PR DESCRIPTION
**Pull request by rachel-fischoff**
## Description

- Created custom icon - a compass icon and named our app "Explore Your World" 
- Users should be able to install the app on their device homsecreens with the custom icon and app name
- Replaced generic react-app-icons with our custom compass favicon, logo.png files & compass.svg
- Added theme color, background color, title and description to app by updating the manifest.json and index.html files 

## Related Issue

Story: Create App Structure 

## Acceptance Criteria

- [x] Once this story is complete, users should be able to install the app on their device homescreens, identifiable as our app, not the generic create-react-app app

- [x] The tags/data points that should be updated include the following:

  - `public/index.html`
  - `<meta name="theme-color">`
  - `<meta name="description">`
  - `<title>`
  - `public/manifest.json`
  - `"short_name"`
  - `"name"`
  -` "icons"`
  - `"theme_color"`
  - `"background_color"`

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
image

### After

image

## Testing Steps / QA Criteria
Locally, you will be able to see the icon and name of the app on the Tab in your browser. Once the branch is approved and deployed, users should be able to install the app on their device homsecreens with the custom icon and app name.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,12 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <title>NearMe</title>
```
Comment (snsebro) [inline public/index.html]: Looks good! Think this might just need to be named in line with whatever we end up naming the app

**Review by snsebro — COMMENTED** [review]
Looks good, added a comment about the Title tag but all good

File: public/index.html (hunk 1 above)
Comment (rachel-fischoff) [reply public/index.html]: I think that's an older file! When you pull up this link and see the files - it looks like it has the correct title! let me know! https://github.com/the-collab-lab/tcl-20-whats-near-me/pull/4/files

**Comment by DeeThompson** [comment]
I agree with Shayne, looks good, all the AC's were met...yay for the favicon :) Looking forward to talking about this on Sunday.

**Comment by segdeha** [comment]
Will look through the code next, but something that would help me as a reviewer would be giving the PR a more descriptive name than "Rf cg app details". Also, you can remove the template instructions from the top of the PR.

image

**Review by segdeha — APPROVED** [review]
This all looks good except the size of your `favicon.ico` image. My understanding, echoed in [this support doc from Microsoft](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/samples/gg491740(v=vs.85)?redirectedfrom=MSDN) (who invented the format), is that a favicon should be max 64x64px.

I've approved the PR, but request that you resize the favicon before merging. Thanks!

**Pull request merged**